webapp/src/components/jira_ticket_tooltip/jira_ticket_tooltip.tsx

```diff
@@ -0,0 +1,277 @@
+        });
+    }
+
+    fixVersionLabel(fixVersion: string) {
+        if (fixVersion) {
+            const fixVersionString = 'Fix Version :';
+            return (
+                <div className='fix-version-label'>
+                    {fixVersionString}
+                    <span className='fix-version-label-value'>
+                        {fixVersion}
+                    </span>
+                </div>
+            );
+        }
+
+        return null;
+    }
+
+    tagTicketStatus(ticketStatus: string) {
+        let ticketStatusClass = 'default-style ticket-status--default';
+
+        const myStatusClass = myStatusClasses[ticketStatus.toLowerCase()];
+        if (myStatusClass) {
+            ticketStatusClass = 'default-style ' + myStatusClass;
+        }
+
+        return <span className={ticketStatusClass}>{ticketStatus}</span>;
+    }
+
+    renderLabelList(labels: string[]) {
+        if (!labels.length) {
+            return null;
+        }
+
+        return (
+            <div className='popover-labels__label'>
+                {
+                    labels.map((label: string, key: number): ReactNode => {
+                        // Return an element for the first three labels and if there are more than three labels, then return a combined label for the remaining labels
+                        if (key < 3 || (key === labels.length - 1 && labels.length > 3)) {
+                            return (
+                                <span
+                                    key={key}
+                                    className='popover-labels__label-list'
+                                >
+                                    {key === labels.length - 1 && labels.length > 3 ? `+${labels.length - 3} more` : label}
+                                </span>);
+                        }
+
+                        return null;
+                    })
+                }
+            </div>
+        );
+    }
+
+    render() {
+        if (!this.state || (!this.state.isLoaded && !this.props.show)) {
+            return (<p/>);
+        }
+
+        const {ticketDetails} = this.state;
+
+        return (
+            <div className='jira-issue-tooltip'>
+                <div className='popover-header'>
+                    <div className='popover-header__container'>
+                        <a
+                            href={this.props.href}
+                            title='Go to ticket'
+                            target='_blank'
+                            rel='noopener noreferrer'
+                        >
+                            <img
+                                src={ticketDetails && ticketDetails.jiraIcon}
+                                width={14}
+                                height={14}
+                                alt={ticketDetails ? 'jira-avatar' : ''}
+                                className={`popover-header__avatar ${!ticketDetails && 'skeleton-loader'}`}
+                            />
```

Comment:
We are adding a classname `skeleton-loader` until the details are not loaded

Reply:
@raghavaggarwal2308 Why use an `img` tag with an invalid `src`, instead of a `div` in replacement of the `img` tag?

Also when `ticketDetails` is defined, we are adding `false` to the class name. We should avoid doing this.